prov/efa/src/efa.h

```diff
@@ -110,6 +110,7 @@
 #define EFA_SHM_MAX_AV_COUNT       (256)
 
 extern int efa_mr_cache_enable;
+extern int efa_system_mem_monitor;
```

Comment:
Can we make `efa_system_mem_monitor` of type `enum efa_mem_monitor_type`? then we can just do `memory_monitors[FI_HMEM_SYSTEM] = efa_system_mem_monitor`